### PR TITLE
Migrate base64encode and base64decode from Bytes to a Base64 module

### PR DESCRIPTION
**Pull request by OceanOak**
Changelog:

```
Stdlib improvements
- Migrate Bytes.base64encode and Bytes.base64decode to a separate Base64 module
- Add Bytes.append and Bytes.containsSegment
```

### CODE REVIEW

File: backend/src/BuiltinExecution/Libs/Bytes.fs [hunk 1]
```diff
@@ -127,6 +64,45 @@ let fns : List<BuiltInFn> =
...
+        (function
+        | _, _, [ DBytes bytes1; DBytes bytes2 ] ->
+          bytes2 |> Array.append bytes1 |> DBytes |> Ply
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplemented
+      previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "containsSegment" 0
```
Comment (OceanOak) [inline backend/src/BuiltinExecution/Libs/Bytes.fs:85]: What do you think about renaming `containsSegment` to just `contains`?

File: backend/src/BuiltinExecution/Libs/Base64.fs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+              |> Ply
+            with e ->
+              Ply(Dval.resultError (DString("Not a valid base64 string")))
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplemented
+      previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "encode" 0
```
Comment (OceanOak) [inline backend/src/BuiltinExecution/Libs/Base64.fs:63]: I am not sure if this should be `urlEncode` instead of `encode`

Comment (pbiggar) [reply backend/src/BuiltinExecution/Libs/Base64.fs:63]: Yeah, I guess that would make sense!

File: backend/src/BuiltinExecution/Libs/Bytes.fs (hunk 1 above)
Comment (pbiggar) [reply backend/src/BuiltinExecution/Libs/Bytes.fs:85]: I prefer `contains` as well. We use `List.contains`, right?

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looks good with those changes!

File: backend/src/BuiltinExecution/Libs/Bytes.fs (hunk 1 above)
Comment (OceanOak) [reply backend/src/BuiltinExecution/Libs/Bytes.fs:85]: I believe we call it `List.member`, we have` String.contains`

**Review by pbiggar — APPROVED** [review]

**Comment by pbiggar** [comment]
I tried solving the conflict with the github conflict editor. I think I got it right? I'll merge once this goes green.

**Pull request merged**